docs/_data/navigation.yml

```diff
@@ -197,3 +139,108 @@ children:
+          - name: terragrunt-iam-role
+            href: /docs/reference/cli-options/#terragrunt-iam-role
+          - name: terragrunt-exclude-dir
+            href: /docs/reference/cli-options/#terragrunt-exclude-dir
+          - name: terragrunt-include-dir
+            href: /docs/reference/cli-options/#terragrunt-include-dir
+          - name: terragrunt-ignore-dependency-order
+            href: /docs/reference/cli-options/#terragrunt-ignore-dependency-order
+          - name: terragrunt-ignore-external-dependencies
+            href: /docs/reference/cli-options/#terragrunt-ignore-external-dependencies
+          - name: terragrunt-include-external-dependencies
+            href: /docs/reference/cli-options/#terragrunt-include-external-dependencies
+          - name: terragrunt-check
+            href: /docs/reference/cli-options/#terragrunt-check
+
+      - name: Built-in Functions
+        href: /docs/reference/built-in-functions/
+        children:
+          - name: Terraform built-in functions
+            href: /docs/reference/built-in-functions/#terraform-built-in-functions
+          - name: find_in_parent_folders
+            href: /docs/reference/built-in-functions/#find_in_parent_folders
+          - name: path_relative_to_include
+            href: /docs/reference/built-in-functions/#path_relative_to_include
+          - name: path_relative_from_include
+            href: /docs/reference/built-in-functions/#path_relative_from_include
+          - name: get_env
+            href: /docs/reference/built-in-functions/#get_env
+          - name: get_terragrunt_dir
+            href: /docs/reference/built-in-functions/#get_terragrunt_dir
+          - name: get_parent_terragrunt_dir
+            href: /docs/reference/built-in-functions/#get_parent_terragrunt_dir
+          - name: get_terraform_commands_that_need_vars
+            href: /docs/reference/built-in-functions/#get_terraform_commands_that_need_vars
+          - name: get_terraform_commands_that_need_input
+            href: /docs/reference/built-in-functions/#get_terraform_commands_that_need_input
+          - name: get_terraform_commands_that_need_locking
+            href: /docs/reference/built-in-functions/#get_terraform_commands_that_need_locking
+          - name: get_terraform_commands_that_need_parallelism
+            href: /docs/reference/built-in-functions/#get_terraform_commands_that_need_parallelism
+          - name: get_aws_account_id
+            href: /docs/reference/built-in-functions/#get_aws_account_id
+          - name: get_aws_caller_identity_arn
+            href: /docs/reference/built-in-functions/#get_aws_caller_identity_arn
+          - name: get_aws_caller_identity_user_id
+            href: /docs/reference/built-in-functions/#get_aws_caller_identity_user_id
+          - name: run_cmd
+            href: /docs/reference/built-in-functions/#run_cmd
+
+      - name: Configuration
+        href: /docs/reference/config-blocks-and-attributes
```

Comment:
I think that missing slash `/` at the end of `href` can be the reason. 

However, I think that you can ignore this. Next PR #1014 will remove `docs/_data/navigation.yml` at all and replace the way how the navigation is being built.